omas/machine_mappings/nstxu.py

```diff
@@ -492,6 +492,48 @@ def mse_data(ods, pulse, MSE_revision="ANALYSIS", MSE_Er_correction=True):
             coef_list.get(f'AA{IMAS2GAM[k]}GAM', [0] * (ch + 1))[ch] for k in range(9)
         ]
 
+# ================================
+@machine_mapping_function(__regression_arguments__, pulse=140001)
+def thomson_scattering_hardware(ods, pulse):
+    """
+    Gathers DIII-D Thomson measurement locations
```

Comment:
DIII-D --> NSTX(-U) here and elsewhere
```suggestion
    Gathers NSTX(-U) Thomson measurement locations
```